packages/toolkit/src/view/settings/user/UserProfileTab.tsx

```diff
@@ -287,47 +309,52 @@ export const UserProfileTab = () => {
                 render={({ field }) => {
                   return (
                     <Form.Item className="w-full">
-                      <FormLabel title="Upload your profile" optional={true} />
+                      <FormLabel
+                        title="Upload your profile test"
+                        optional={true}
+                      />
                       <Form.Control>
-                        <label
-                          htmlFor="org-logo-input"
-                          className="flex h-[150px] w-full cursor-pointer flex-col items-center justify-center rounded border border-dashed border-semantic-bg-line bg-semantic-bg-base-bg text-semantic-fg-secondary product-body-text-3-medium"
-                        >
-                          {field.value ? (
-                            <img
-                              src={field.value}
-                              alt={`${user.data?.name}-profile`}
-                              className="h-[150px] w-full object-contain"
-                            />
-                          ) : (
-                            <p>Upload your profile</p>
-                          )}
-
-                          <Input.Root className="hidden">
-                            <Input.Core
-                              {...field}
-                              id="org-logo-input"
-                              type="file"
-                              accept="images/*"
-                              value={undefined}
-                              onChange={async (e) => {
-                                const file = e.target.files?.[0];
-
-                                if (file) {
-                                  const reader = new FileReader();
-
-                                  reader.onload = () => {
-                                    const result = reader.result;
-
-                                    field.onChange(result);
-                                  };
-
-                                  reader.readAsDataURL(file);
+                        <div>
+                          <label
+                            htmlFor="org-logo-input"
+                            className="flex h-[150px] w-full cursor-pointer flex-col items-center justify-center rounded border border-dashed border-semantic-bg-line bg-semantic-bg-base-bg text-semantic-fg-secondary product-body-text-3-medium"
+                          >
+                            {field.value ? (
+                              <img
+                                src={
+                                  profileAvatar ? profileAvatar : field.value
                                 }
-                              }}
-                            />
-                          </Input.Root>
-                        </label>
+                                alt={`${user.data?.name}-profile`}
+                                className="h-[150px] w-full object-contain"
```

Comment:
Due to we want to display the rounded corner version. we can write as 

```suggestion
                                alt={`${user.data?.name}-profile`}
                                className="h-[150px] rounded-full object-contain"
```